### ISSUE TITLE
Arguments totally broken

### ISSUE THREAD

**Issue by qiemem** [arthurhjorth/LevelSpaceXWGUI#37]
First, command arguments are appearing where agent arguments should be.

Next, whenever you try to save a relationship with command arguments, you get the following exception (`amount` is the name of the argument).

```
NetLogo is unable to supply you with more details about this error.  Please report the problem
at https://github.com/NetLogo/NetLogo/issues, or to bugs@ccl.northwestern.edu, and paste the
contents of this window into your report.

java.util.NoSuchElementException: key not found: amount
 at scala.collection.MapLike$class.default(MapLike.scala:225)
 at scala.collection.immutable.Map$EmptyMap$.default(Map.scala:88)
 at scala.collection.MapLike$class.apply(MapLike.scala:135)
 at scala.collection.immutable.Map$EmptyMap$.apply(Map.scala:88)
 at org.levelspace.LSArgumentSelector$$anonfun$selectedArgumentIndices_$eq$1.apply(LSArgumentSelector.scala:31)
 at scala.collection.Iterator$class.foreach(Iterator.scala:772)
 at scala.collection.immutable.VectorIterator.foreach(Vector.scala:648)
 at org.levelspace.LSArgumentSelector.selectedArgumentIndices_$eq(LSArgumentSelector.scala:27)
 at org.levelspace.RelationshipKind$$anonfun$42.apply(RelationshipWidget.scala:77)
 at org.levelspace.RelationshipKind$$anonfun$42.apply(RelationshipWidget.scala:77)
 at uk.ac.surrey.xw.api.Property$$anonfun$set$1.apply(Property.scala:37)
 at uk.ac.surrey.xw.api.Property$$anonfun$set$1.apply(Property.scala:37)
 at scala.Option.foreach(Option.scala:197)
 at uk.ac.surrey.xw.api.Property.set(Property.scala:37)
 at uk.ac.surrey.xw.api.ExtraWidget$$anonfun$setProperty$1.apply(ExtraWidget.scala:26)
 at uk.ac.surrey.xw.api.ExtraWidget$$anonfun$setProperty$1.apply(ExtraWidget.scala:25)
 at scala.Option.foreach(Option.scala:197)
 at uk.ac.surrey.xw.api.ExtraWidget$class.setProperty(ExtraWidget.scala:25)
 at org.levelspace.Relationship.setProperty(RelationshipWidget.scala:107)
 at uk.ac.surrey.xw.gui.GUI$$anonfun$uk$ac$surrey$xw$gui$GUI$$setProperty$1.apply(GUI.scala:76)
 at uk.ac.surrey.xw.gui.GUI$$anonfun$uk$ac$surrey$xw$gui$GUI$$setProperty$1.apply(GUI.scala:76)
 at scala.Option.foreach(Option.scala:197)
 at uk.ac.surrey.xw.gui.GUI.uk$ac$surrey$xw$gui$GUI$$setProperty(GUI.scala:75)
 at uk.ac.surrey.xw.gui.GUI$$anonfun$notify$2.apply$mcV$sp(GUI.scala:49)
 at uk.ac.surrey.xw.gui.GUI$$anonfun$notify$2.apply(GUI.scala:49)
 at uk.ac.surrey.xw.gui.GUI$$anonfun$notify$2.apply(GUI.scala:49)
 at uk.ac.surrey.xw.api.package$$anon$1.run(package.scala:28)
 at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
 at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:715)
 at java.awt.EventQueue.access$400(EventQueue.java:82)
 at java.awt.EventQueue$2.run(EventQueue.java:676)
 at java.awt.EventQueue$2.run(EventQueue.java:674)
 at java.security.AccessController.doPrivileged(Native Method)
 at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:86)
 at java.awt.EventQueue.dispatchEvent(EventQueue.java:685)
 at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
 at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
 at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
 at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
 at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
 at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)

NetLogo 5.2.0
main: org.nlogo.lite.InterfaceComponent
thread: AWT-EventQueue-0
Java HotSpot(TM) 64-Bit Server VM 1.6.0_65 (Apple Inc.; 1.6.0_65-b14-462-11M4609)
operating system: Mac OS X 10.9.5 (x86_64 processor)
Scala version 2.9.2
JOGL: (3D View not initialized)
OpenGL Graphics: (3D View not initialized)
model: /Users/bryan/Development/NetLogo/lsxwgui/Wolf Sheep Predation.nlogo

08:52:23.267 JobRemovedEvent (org.nlogo.app.App$$anon$1 (org.nlogo.window.GUIWorkspace)) JobThread
08:52:23.267 InterfaceGlobalEvent (org.nlogo.window.SliderWidget) AWT-EventQueue-0
08:52:23.267 InterfaceGlobalEvent (org.nlogo.window.SliderWidget) AWT-EventQueue-0
08:52:23.267 InterfaceGlobalEvent (org.nlogo.window.SliderWidget) AWT-EventQueue-0
08:52:23.267 InterfaceGlobalEvent (org.nlogo.window.SliderWidget) AWT-EventQueue-0
08:52:23.266 InterfaceGlobalEvent (org.nlogo.window.SliderWidget) AWT-EventQueue-0
08:52:23.266 InterfaceGlobalEvent (org.nlogo.window.SliderWidget) AWT-EventQueue-0
08:52:23.266 InterfaceGlobalEvent (org.nlogo.widget.SwitchWidget) AWT-EventQueue-0
08:52:23.266 InterfaceGlobalEvent (org.nlogo.window.SliderWidget) AWT-EventQueue-0
08:52:23.266 InterfaceGlobalEvent (org.nlogo.widget.SwitchWidget) AWT-EventQueue-0
```

**Comment by arthurhjorth** [comment]
this is fixed on the save branch and will be merged either tonight or tomorrow early morning

**Comment by arthurhjorth** [comment]
fixed with #38
